tests/admin_checks/tests.py

```diff
@@ -43,6 +45,11 @@ def check(self, **kwargs):
 )
 class SystemChecksTestCase(SimpleTestCase):
 
+    def setUp(self):
+        super(SystemChecksTestCase, self).setUp()
+        admin.sites.all_sites.clear()
```

Comment:
I think the setup/teardown is obsoleted by my PR only running checks for models in installed apps